amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -67,6 +67,15 @@ function submitClean(){
 	document.aimAuditLoggerManagerForm.submit();
 }
 
+function submitFilter() {
+	document.aimAuditLoggerManagerForm.selectedUser.value = document.getElementById("userId").value;

```

Comment:
You dont need to do this, selected value already has the value chosen from the dropdown